packages/shared/src/types/abTests/shared.ts

```diff
@@ -64,3 +64,10 @@ export interface PurchaseInfoTest {
     userType: purchaseInfoUser[];
     product: purchaseInfoProduct[];
 }
+
+export interface PageContextTargeting {
+    tagIds: string[]; // tags must include one of these
+    sectionIds: string[]; // section must be one of these
+    excludedTagIds: string[]; // tags must not include one of these
+    excludedSectionIds: string[]; // section must not be one of these
```

Comment:
Until I read the code/tests I wasn't 100% sure if the context matches were ORed or ANDed together, I wonder if making that clear in the comments here would be helpful?

```suggestion
    tagIds: string[]; // tags must include one of these
    sectionIds: string[]; // AND section must be one of these
    excludedTagIds: string[]; // AND tags must not include one of these
    excludedSectionIds: string[]; // AND section must not be one of these
```

What do you think?